domain/src/main/java/com/lcw/study/nextstep/domain/OperationType.kt

```diff
@@ -0,0 +1,23 @@
+package com.lcw.study.nextstep.domain
+
+enum class OperationType {
+    PLUS,
+    MINUS,
+    MULTIPLY,
+    DIVIDE,
+    NOTOPERATIONTYPE;
```

Comment:
```suggestion
    NOT_OPERATION_TYPE;
```
하이픈을 활용해주시면 의미하는 바를 이해하기 쉬워질 것 같아요.

Reply:
758471e 수정하였습니다!